Scheduler/observing_blocks_maintel/BLOCK-T189.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "BLOCK-T189",
+    "program": "BLOCK-T189",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/take_image_lsstcam.py",
+            "standard": true,
+            "parameters": {
+                "nimages": "$num_exp",
+                "exp_times": "$exp_times",
+                "image_type": "ENGTEST",
+                "reason": "BLOCK-T189",
+                "program": "$run_id",
```

Comment:
How are you expecting to pass these?

Reply:
It looked to me it is not possible to pass it. I will remove to avoid confusion.